app/models/host_aggregate.rb

```diff
@@ -0,0 +1,43 @@
+class HostAggregate < ApplicationRecord
+  include NewWithTypeStiMixin
+  include Metric::CiMixin
+  include EventMixin
+  include ProviderObjectMixin
+  include VirtualTotalMixin
+
+  acts_as_miq_taggable
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+
+  # FIXME: I don't think this is needed here
```

Comment:
You don't have the rollup defined here, so not sure what is wanted.

Vm's metrix are collected in cloud and Host's metrics are collected in infra. I don't know if your rollup will be based on Hosts or VMs.

Possibly, we should have 2 Hosts metrics in the end. One obtained directly from infra, as hardware metrics. And one as a Vm's metrics rollup for host.